packages/server-wallet/src/__test__/create-and-close-channel/ledger-funding.test.ts

```diff
@@ -608,11 +608,11 @@ describe('Funding multiple channels syncronously without enough funds', () => {
 
     expect(unfundedChannels).toMatchObject([
       {
-        turnNum: 1,
+        turnNum: 0,
```

Comment:
Currently both A and B sign 0, and then both sign 3. It should be easy to change if we wanted to do 0,1,2,3 but I know there have been some discussions around this, and wanted to make it consistent between ledger funding and direct funding.

Reply:
I've realised that the pre-existing behaviour was consistend, so I've changed it back to incrementing turnNums in all cases.